docs/source/usage/workflows/ionization.rst

```diff
@@ -0,0 +1,187 @@
+.. _how_to_setup_ionization:
+
+Ionization
+==========
+(See :ref:`usage-params` for how to configure a picongpu simulation in general)
+
+.. note::
+    The `default ionizer.param <https://github.com/ComputationalRadiationPhysics/picongpu/blob/dev/include/picongpu/param/ionizer.param>`_ and `default ionizationEnergies.param <https://github.com/ComputationalRadiationPhysics/picongpu/blob/dev/include/picongpu/param/ionizationEnergies.param>`_ includes all atomic parameters for some species.
+
+To include charge-state-only simulations in your simulations you must:
+
+1. Define the atomic parameters of all ionization ion species of your simulation in the :code:`ionizer.param`.
+
+    1.) number of protons and neutrons
+
+        .. code-block:: c++
+
+            /* Example: nitrogen */
+            namespace picongpu::ionization::atomicNumbers
+            {
+                struct Nitrogen
+                {
+                    static constexpr float_X numberOfProtons  = 7.0;
+                    static constexpr float_X numberOfNeutrons = 7.0;
+                };
+            } // namespace picongpu::ionization::atomicNumbers
+
+    2.) effective nuclear charge for each charge state
+
+        .. code-block:: c++
+
+            /* Example: nitrogen */
+            namespace picongpu::ionization::effectiveNuclearCharge
+            {
+                /// @attention insert the values in REVERSE order since the lowest shell corresponds to the last ionization process.
+                PMACC_CONST_VECTOR(
+                    float_X,
+                    7, // number of charge states
+                    Nitrogen, // name you may reference this by later, remember to prepend the namespace!
+                    /* 2p^3 */
+                    3.834,
+                    3.834,
+                    3.834,
+                    /* 2s^2 */
+                    3.874,
+                    3.874,
+                    /* 1s^2 */
+                    6.665,
+                    6.665);
+            } // namespace picongpu::ionization::effectiveNuclearCharge
+
+        .. note::
+            see `wikipedia <https://en.wikipedia.org/wiki/Effective_nuclear_charge>`_ for values or refer directly to the calculations by Slater or Clementi and Raimondi, [1]_ and [2]_.
```

Comment:
Neither reference 1 nor 2 is a "Slater" paper. Both are Clementi and Raimondi.

Reply:
this was copied from the FoilLTC example, removed the reference to Slater,